internal/config/releases.go

```diff
@@ -70,7 +70,14 @@ func getLatestReleases(releases uint) ([]*Release, error) {
 	releaseAPIURL := "https://api.github.com/repos/linkerd/linkerd2/releases?per_page=" + fmt.Sprint(releases)
 	// We need a variable url here hence using nosec
 	// #nosec
-	resp, err := http.Get(releaseAPIURL)
+	client := &http.Client{}
+
+	req, err := http.NewRequest("GET", releaseAPIURL, http.NoBody)
```

Comment:
Is there I have considered the suggestion from `golangci-lint`?

`should rewrite http.NewRequestWithContext or add (*Request).WithContext (noctx)`